docs/markdown/Cross-compilation.md

```diff
@@ -10,17 +10,23 @@ nomenclature. The three most important definitions are traditionally
 called *build*, *host* and *target*. This is confusing because those
 terms are used for quite many different things. To simplify the issue,
 we are going to call these the *build machine*, *host machine* and
-*target machine*. Their definitions are the following
+*target machine*. Their definitions are the following:
 
-* *build machine* is the computer that is doing the actual compiling
-* *host machine* is the machine on which the compiled binary will run
-* *target machine* is the machine on which the compiled binary's output will run (this is only meaningful for programs such as compilers that, when run, produce object code for a different CPU than what the program is being run on)
+* *build machine* is the computer that is doing the actual compiling.
+* *host machine* is the machine on which the compiled binary will run.
+* *target machine* is the machine on which the compiled binary's
+  output will run, *only defined* if the program produces
```

Comment:
Not sure why you changed 'meaningful' to 'defined' here.

This could be taken as a statement that the `target_machine` object is undefined in some cases, whereas it's just not useful to use it.

Reply:
No reason, that was the incidental result of a few cumulative rewrites per the [above request](https://github.com/mesonbuild/meson/pull/3308#discussion_r177201758) to get rid of parentheses here. I'll change back